.NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestDateParser.cs

```diff
@@ -186,5 +248,33 @@ public void TestDateParseLuis()
             BasicTest("I'll go back on Friday next week", "2016-11-18");
 
         }
+
+        [TestMethod]
+        public void TestDateParseForTheLuis()
+        {
+            BasicTest("I went back for the 27", "XXXX-XX-27");
+            BasicTest("I went back for the 27th", "XXXX-XX-27");
+            BasicTest("I went back for the 27.", "XXXX-XX-27");
+            BasicTest("I went back for the 27!", "XXXX-XX-27");
+            BasicTest("I went back for the 27 .", "XXXX-XX-27");
+            BasicTest("I went back for the 21st", "XXXX-XX-21");
+            BasicTest("I went back for the 22nd", "XXXX-XX-22");
+            BasicTest("I went back for the second", "XXXX-XX-02");

```

Comment:
Same.

Reply:
The parser will get no input if it is not extracted. Adding the negative cases in the extractors is enough.